reference/constraints/Unique.rst

```diff
@@ -123,4 +125,18 @@ Parameter                      Description
 ``{{ value }}``                The current (invalid) value
 =============================  ================================================
 
+``normalizer``
+~~~~~~~~~~~~~~
+
+**type**: a `PHP callable`_ **default**: ``null``
+
+This option allows to define the PHP callable applied to each element of the given collection before
+checking if the collection is valid.
+
+For example, you may want to pass the ``'trim'`` string to apply the
+:phpfunction:`trim` PHP function to each element of the collection in order to ignore leading and trailing
+whitespace during validation.
+
```

Comment:
Please add a versionadded directive

The ''normalizer'' option was introduced in Symfony 5.3.

' is a backtick, I am currently on a phone 📱😄